ruby/classes_and_objects/lib/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
+  attr_accessor :name, :account_no, :balance
+  @@total_accounts = 0
+
+  def initialize(input_name, initial_balance)
+    @name = input_name
+    @balance = initial_balance
+    @@total_accounts += 1
+    @account_no = @@total_accounts
+  end
+
+  def transaction(transfer_amount, depositer)
```

Comment:
There are two types of transactions, credit and debit. What is happening here?

Reply:
sir both transaction happening here. first it debit  then credit.